lighthouse-core/report/v2/report-renderer.js

```diff
@@ -64,15 +71,31 @@ window.ReportRenderer = class ReportRenderer {
   }
 
   /**
-   * @param {!ReportJSON} report
-   * @return {!Element}
+   * Sets the text content of an element, safely. If the element does not exist,
+   * results in a noop.
+   * @param {Element} element
+   * @param {!string} text
+   * @return {Element}
    */
-  renderReport(report) {
-    try {
-      return this._renderReport(report);
-    } catch (e) {
-      return this._renderException(e);
+  static setText(element, text) {
+    if (element) {
```

Comment:
@paulirish says, "I am low stress on this. Do whateva :)"

Note to future me: give PI 24hrs and he will be low stress.